dune-project

```diff
@@ -2,7 +2,7 @@
 (name goblint-cil)
 (implicit_transitive_deps false)
 (generate_opam_files true)
-(version 2.0.5)
+(version 2.0.6)
```

Comment:
The version number should remain the same for the time being. It's not a patch-level change either.

Reply:
I think I want to make a last 2.0.6 release without this and have this in 2.1.0.